prov/efa/src/rxr/rxr_pkt_type_req.c

```diff
@@ -123,11 +123,11 @@ size_t rxr_pkt_req_data_size(struct rxr_pkt_entry *pkt_entry)
 	return pkt_entry->pkt_size - rxr_pkt_req_data_offset(pkt_entry);
 }
 
-int rxr_pkt_type_readbase_rtm(struct rdm_peer *peer, int op, uint64_t fi_flags)
+int rxr_pkt_type_readbase_rtm(struct rdm_peer *peer, int op, uint64_t fi_flags, struct efa_hmem_info *hmem_info)
```

Comment:
could u add a doxygen docstring for this function.

Reply:
done